### PR TITLE
Replace a few refresher lesson links with concept card tags, instead

### PR DESCRIPTION
**Pull request by BenHenning**

### CODE REVIEW

File: domain/src/main/assets/fractions_exploration1.json [hunk 1]
```diff
@@ -3933,7 +3933,7 @@
               "param_changes": [],
               "feedback": {
                 "content_id": "feedback_4",
-                "html": "<p>No, that's not right.</p><p>It looks like you've forgotten what the numerator and denominator represent. You should probably try the <oppia-noninteractive-link text-with-value=\"&amp;quot;refresher lesson&amp;quot;\" url-with-value=\"&amp;quot;https://www.oppia.org/explore/xHwiGtvK2N4L?parent=MjZzEVOG47_1&amp;quot;\"></oppia-noninteractive-link> to refresh your memory.<br></p>"
+                "html": "<p>No, that's not right.</p><p>It looks like you've forgotten what the numerator and denominator represent. You should probably try the <oppia-concept-card-link skill-id=\"UxTGIJqaHMLa\">refresher lesson</oppia-concept-card-link> to refresh your memory.<br></p>"
```
Comment (seanlip) [inline domain/src/main/assets/fractions_exploration1.json]: I think the text should be: Take a look at this [concept card] to refresh your memory.

File: domain/src/main/assets/fractions_exploration1.json [hunk 2]
```diff
@@ -5304,7 +5304,7 @@
               "param_changes": [],
               "feedback": {
                 "content_id": "feedback_6",
-                "html": "<p>No, that's not right. It looks like you've forgotten what the numerator and denominator represent! The denominator should be the <strong>total</strong> <strong>number of equal parts</strong>, but it isn't in your answer.</p><p>You should probably try the <oppia-noninteractive-link text-with-value=\"&amp;quot;refresher lesson&amp;quot;\" url-with-value=\"&amp;quot;https://www.oppia.org/explore/xHwiGtvK2N4L?parent=MjZzEVOG47_1&amp;quot;\"></oppia-noninteractive-link> to refresh your memory.</p>"
+                "html": "<p>No, that's not right. It looks like you've forgotten what the numerator and denominator represent! The denominator should be the <strong>total</strong> <strong>number of equal parts</strong>, but it isn't in your answer.</p><p>You should probably try the <oppia-concept-card-link skill-id=\"UxTGIJqaHMLa\">refresher lesson</oppia-concept-card-link> to refresh your memory.</p>"
```
Comment (seanlip) [inline domain/src/main/assets/fractions_exploration1.json]: Replace last sentence with: If you like, you can take a look at this [concept card] to refresh your memory.

**Review by seanlip — CHANGES_REQUESTED** [review]

File: domain/src/main/assets/fractions_exploration1.json (hunk 1 above)
Comment (BenHenning) [reply domain/src/main/assets/fractions_exploration1.json]: Done.

File: domain/src/main/assets/fractions_exploration1.json (hunk 2 above)
Comment (BenHenning) [reply domain/src/main/assets/fractions_exploration1.json]: Done.

**Review by seanlip — APPROVED** [review]
Thanks!

**Pull request merged**